src/widgets/Widget.ts

```diff
@@ -202,7 +202,7 @@ export default abstract class Widget {
     }
 
     let found: Element | null = null;
-    for (let i = 0; i < 50; i++) {
+    for (let i = 0; i < 5; i++) {
```

Comment:
@00salmon we likely only need to do kind of polling in qwirks mode. If it's standards mode, then we can likely just use `document.body` without any need for polling.

Related, here is the MDN article on how to detect if the iframe is in standards or qwirks mode: https://developer.mozilla.org/en-US/docs/Web/API/Document/compatMode

```suggestion
    for (let i = 0; i < 5; i++) {
```